CHANGELOG.md

```diff
@@ -21,7 +21,10 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.8.0...master
+[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.8.1...master
+
+## [2.8.1]
```

Comment:
I think it should be a minor version release since we added `ApacheEc2LoadBalancerFormula` to API, not a patch release.

Reply:
Done.